src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/OpenSslCachedSystemStoreProvider.cs

```diff
@@ -363,23 +368,26 @@ private static string[] GetRootStoreDirectories(out bool isDefault)
         }
 
         private static bool TryStatFile(string path, out DateTime lastModified)
-            => TryStat(path, Interop.Sys.FileTypes.S_IFREG, out lastModified);
+            => CallStat(path, Interop.Sys.FileTypes.S_IFREG, out lastModified, out _);
 
         private static bool TryStatDirectory(string path, out DateTime lastModified)
-            => TryStat(path, Interop.Sys.FileTypes.S_IFDIR, out lastModified);
+            => CallStat(path, Interop.Sys.FileTypes.S_IFDIR, out lastModified, out _);
 
-        private static bool TryStat(string path, int fileType, out DateTime lastModified)
+        private static bool TryStatFile(string path, out DateTime lastModified, out (long, long) fileId)
```

Comment:
nit: only keep 1 `TryStatFile` and update the caller that ignores `fileId` by passing it `out _` as the last argument.